test_base_fileurl_field/models/res_partner.py

```diff
@@ -21,24 +21,32 @@ class ResPartner(models.Model):
 
     @api.constrains("url_file", "url_file_fname")
     def _check_url_file_fname(self):
-        rec = self.search([("url_file_fname", "=", self.url_file_fname)])
-        if len(rec) > 1:
-            raise ValidationError(
-                _(
-                    "This file name is already used on an existing record. "
-                    "Please use another file name or delete the url_file on :\n"
-                    "Model: %s Id: %s" % (self._name, rec.id)
-                )
+        for rec in self:
+            match = self.search(
```

Comment:
would be better to use `search_count` in such cases

Reply:
Outside of topic of this PR I'm only applying lint and making CI happy.